build.gradle

```diff
@@ -118,6 +118,8 @@ dependencies {
     }
     compile 'org.springframework.hateoas:spring-hateoas:0.19.0.RELEASE'
 
+    compile 'org.eclipse.jetty:jetty-servlets:9.4.0.M0'
```

Comment:
Off course, I just modified to `9.2.14.v20151106` like in the Scheduling project, it is safer this way, thanks !